modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro

```diff
@@ -100,6 +100,14 @@ definition {
 			locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT");
 	}
 
+	macro assertNoColumnAddYet {
```

Comment:
Can you change this to assertNoColumnPresentInDefaultSort to make it clearer what we're asserting?